{"name": "创建英雄失败的测试用例-攻击力不符合要求", "status": "passed", "parameters": [{"name": "success_for_name", "value": "'123@qq.com'"}, {"name": "success_for_volume", "value": "2"}, {"name": "fail_for_power", "value": "-50"}], "start": 1684150563237, "stop": 1684150563237, "uuid": "64af75d6-f099-4741-bf74-6a608dd50555", "historyId": "930bbc126d70e9a42eb3e2ffc779c41b", "testCaseId": "052da0c40443119f715fb4201d430dc8", "fullName": "test_create_hero_homework_01.TestHeroCreate#test_create_hero_fail_for_power", "labels": [{"name": "tag", "value": "run(order=4)"}, {"name": "suite", "value": "test_create_hero_homework_01"}, {"name": "subSuite", "value": "TestHeroCreate"}, {"name": "host", "value": "LAPTOP-0EHG7NSH"}, {"name": "thread", "value": "48660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_hero_homework_01"}]}